### PR TITLE
Remove action property `modal` default value.

### PR DESCRIPTION
**Pull request by petschki**
fixes https://github.com/plone/Products.CMFPlone/issues/3801

### CODE REVIEW

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

File: news/3801.bugfix [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Remove action property `modal` default value.
```
Comment (thet) [inline news/3801.bugfix:1]:
```suggestion
Remove action property `modal` default value.
Fixes: https://github.com/plone/Products.CMFPlone/issues/3801
```

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

**Review by mauritsvanrees — APPROVED** [review]
Thanks!  Checked, and it works.

There *is* something still not exactly right though, but that is not caused by this PR.
When I edit and save the action in the Plone UI, and then go to `portal_actions` in the ZMI, both the `modal` and `condition` fields show the text `None` as value.  If you then save the form in the ZMI, the action is no longer shown, because the literal text `None` is not a true condition.
I wondered if using `default=""` would help, but then Plone does not even start up, because the modal field has a json validator that then fails.

But as long as you stay in the Plone UI, it is fine.  And this is really a different issue.

So: approved, thanks!

**Pull request merged**